packages/artifact/src/internal/utils.ts

```diff
@@ -296,3 +304,11 @@ export async function createDirectoriesForArtifact(
     })
   }
 }
+
+export async function createEmptyFilesForArtifact(
```

Comment:
I followed the guidance [here](https://stackoverflow.com/questions/12809068/create-an-empty-file-in-node-js) on how to create an empty file in node.